framework/16-inch/cpu/7040-amd/README.md

```diff
@@ -0,0 +1,18 @@
+# [Framework Laptop 16](https://frame.work/)
+
+## Updating Firmware
```

Comment:
I think this section could be better shortened to something like "The Framework Laptop 16 uses LVFS, so it can be updated via fwupd; see https://wiki.nixos.org/wiki/Fwupd for details"

Both the instructions on here and those on the wiki are pretty short, but the wiki is the canonical source.

The link to the LVFS page is not directly useful, since you'd normally do it via fwupd.